sound/soc/sof/intel/Makefile

```diff
@@ -18,7 +16,6 @@ snd-sof-intel-atom-objs := atom.o
 obj-$(CONFIG_SND_SOC_SOF_INTEL_ATOM_HIFI_EP) += snd-sof-intel-atom.o
 obj-$(CONFIG_SND_SOC_SOF_BAYTRAIL) += snd-sof-acpi-intel-byt.o
 obj-$(CONFIG_SND_SOC_SOF_BROADWELL) += snd-sof-acpi-intel-bdw.o
-obj-$(CONFIG_SND_SOC_SOF_INTEL_HIFI_EP_IPC) += snd-sof-intel-ipc.o
```

Comment:
Can you also remove other traces of SND_SOC_SOF_INTEL_HIFI_EP_IPC from sof/intel/Kconfig and bdw/byt/pci-tng no longer needs the MODULE_IMPORT_NS(SND_SOC_SOF_INTEL_HIFI_EP_IPC) as it is gone.